articles/integrations/azure-api-management/configure-azure.md

```diff
@@ -0,0 +1,172 @@
+
+## Configure Azure
+
+::: note
+To complete this, tutorial, you will need to have an account that grants you access to Microsoft's [Azure Portal](https://portal.azure.com).
+:::
+
+In this section, you'll begin by creating an API management instance, as well as the API to manage your cookies application.
+
+### Create Your API Management Instance
```

Comment:
on the previous doc you had steps in your headers, I'd add them here as well `Step 1: Create Your API Management Instance`